packages/hdwallet-native/src/crypto/isolation/engines/default/bip32.ts

```diff
@@ -90,14 +90,7 @@ export class Node extends Revocable(class {}) implements BIP32.Node, SecP256K1.E
     const entropy = counter === undefined ? undefined : Buffer.alloc(32);
     entropy?.writeUInt32BE(counter ?? 0, 24);
     return await SecP256K1.RecoverableSignature.fromSignature(
-      checkType(
-        SecP256K1.Signature,
-        (
-          tinyecc as typeof tinyecc & {
-            signWithEntropy: (message: Buffer, privateKey: Buffer, entropy?: Buffer) => Buffer;
-          }
-        ).signWithEntropy(Buffer.from(msgOrDigest), this.#privateKey, entropy)
-      ),
+      checkType(SecP256K1.Signature, ecc.sign(Buffer.from(msgOrDigest), this.#privateKey, entropy)),
```

Comment:
- same as before without the cast
- `signEntropy` is now renamed to `sign` but is the same method, see `sign` signature on v2 https://www.npmjs.com/package/tiny-secp256k1#sign-h-d-e vs `signWithEntryopy` signature on v1 https://www.npmjs.com/package/tiny-secp256k1/v/1.1.7#signWithEntropy-h-d-e